packages/datadog-plugin-rhea/src/index.js

```diff
@@ -1,73 +1,92 @@
+'use strict'
+
+let CircularBuffer
+let popIf
+
+function patchCircularBuffer (CircularBuffer) {
```

Comment:
Can this be rewritten in the standard way? (i.e. wrapCircularBuffer or createWrapCircularBuffer)

Reply:
Also, small nit but keeping the main wrapper factories at the top of the function makes it easier to find the entrypoint of the instrumentation.

Reply:
We don't have CircularBuffer when statically loading the library like we do with other classes, because it isn't exported. Therefore the wrapping needs to happen the first time we see the class. That being said, I think I can refactor this to use `this.wrap`.

Reply:
Yes I just meant to use a syntax similar to the other wrapper functions, a bit as if it was actually available. This makes it easier to read since it's similar to the other, and if CircularBuffer is ever exposed it would be a one line change.